samples/websocket_connect.md

```diff
@@ -4,6 +4,12 @@
 
 This sample makes an MQTT connection via Websockets and then disconnects. On startup, the device connects to the server via Websockets and then disconnects right after. This sample is for reference on connecting via Websockets. This sample demonstrates the most straightforward way to connect via Websockets by querying the AWS credentials for the connection from the device's environment variables or local files.
 
+If you want to use custom auth (or static creds, or basic auth, etc) instead,
+then you will need to replace part of the sample (connection\_setup function) with a code snippet we provided in its corresponding readme.
+
+* [custom auth](./websocket_connect_custom_auth.md)
```

Comment:
custom auth and static credentials label should also explicitly be named with websocket included. The links are to the websocket md files but the link text on display could be confusing/misleading.